contractsV5/KyberNetwork.sol

```diff
@@ -85,18 +73,22 @@ contract KyberNetwork is Withdrawable, Utils, IKyberNetwork, ReentrancyGuard {
     {
         TradeData memory tradeData;
 
-        tradeData.input.trader = address(uint160(trader));
-        tradeData.input.src = src;
-        tradeData.input.srcAmount = srcAmount;
-        tradeData.input.dest = dest;
-        tradeData.input.destAddress = address(uint160(destAddress));
-        tradeData.input.maxDestAmount = maxDestAmount;
-        tradeData.input.minConversionRate = minConversionRate;
-        tradeData.input.platformWallet = walletId;
-        tradeData.input.platformFeeBps = 0;
+        initTradeData(
```

Comment:
actually the above is better. since its safer.

if using this one
please use the set parameter by name convention...
same for all initTradeData  calls.

Reply:
It means duplicate code, but if that's safer, ok.